src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -302,7 +302,7 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
     [IsDesignScriptCompatible]
     public class FloorTypes : RevitDropDownBase
     {
-        private const string noFloorTypes = "No floor types available.";
+        private string noFloorTypes = Properties.Resources.NoFloorTypesAvailable;
```

Comment:
Replace `noFloorTypes` with resource string, it is not necessary to keep this field there.